blkn/src/InputText/index.js

```diff
@@ -9,23 +10,12 @@ type Props = {
   error?: string,
 };
 
-const Input = (props: Props) => (
+const InputText = (props: Props) => (
   <div>
```

Comment:
why are you splashing `div`s everywhere? 😄

Reply:
I re-used existing code and didn't change its HTML structure. :-)